local-cli/runIOS/runIOS.js

```diff
@@ -129,7 +133,15 @@ function buildProject(xcodeProject, udid, scheme) {
     '-derivedDataPath', 'build',
   ];
   console.log(`Building using "xcodebuild ${xcodebuildArgs.join(' ')}"`);
-  child_process.spawnSync('xcodebuild', xcodebuildArgs, {stdio: 'inherit'});
+  var buildProcess = child_process.spawnSync('xcodebuild', xcodebuildArgs, {stdio: 'pipe'});
+  var buildOutput = buildProcess.stdout.toString();
+  console.log(buildOutput);
```

Comment:
I had the same concern.  I tried different combinations for stdout and got nothing that would pipe output and allow me to buffer it.  I totally get the problem it creates but node basically gives you the option to buffer or pipe: https://nodejs.org/api/child_process.html#child_process_child_process_spawnsync_command_args_options

Another option could be to do try using child_process.spawn, as done [here](http://stackoverflow.com/questions/10232192/exec-display-stdout-live).

I'll see what I can do.  But IMO it's worth it now, considering it is fixing a bug that I consider to be major.  Even adding a console.log("Building, please wait this can take a while." before could be good enough for now.